src/execution/execute.js

```diff
@@ -1051,10 +1052,48 @@ function completeListValue(
     // No need to modify the info object containing the path,
     // since from here on it is not ever accessed by resolver functions.
     const fieldPath = addPath(path, index);
+    const initialFieldNodes = [];
+    for (const fieldNode of fieldNodes) {
+      const stream = getDirectiveValues(
+        GraphQLStreamDirective,
+        fieldNode,
+        exeContext.variableValues,
+      );
+      if (
+        exeContext.schema.__experimentalStream &&
+        stream &&
+        stream.if !== false &&
+        typeof stream.label === 'string' &&
+        typeof stream.initial_count === 'number' &&
```

Comment:
I don't think you need the type checking here since the compiler should catch incorrect types.

Reply:
you do need it for the flow checks to pass, you had to do the same thing here: https://github.com/lilianammmatos/graphql-js-experimental/blob/2573d535e893e480dca59f55e9eeea82c02fb3ba/src/execution/execute.js#L660